docs/docs/architecture/attestation.md

```diff
@@ -144,7 +144,7 @@ The latter means that the value can be generated offline and compared to the one
 | 7           | Secure Boot State                                                | Azure, Constellation Bootloader | No                          |
 | 8           | -                                                                | -                               | -                           |
 | 9           | initramfs                                                        | Linux Kernel                    | Yes                         |
-| 10          | Reserved                                                         | -                               | No                          |
+| 10          | Userspace[^1]                                                    | Linux IMA[^1]                   | No[^1]                      |
```

Comment:
nit: I have a slight preference for adding the reference only in the last column. But keep it as is if you prefer.